backend/src/docs/v1.api-spec.yaml

```diff
@@ -299,6 +329,57 @@ components:
           type: string
           description: 'Contains the service client name that was written to WebADE'
           example: DOMO_SERVICE_CLIENT
+    KcConfigForm:
+      type: object
+      required:
+        - applicationAcronym
+        - applicationName
+        - applicationDescription
+        - commonServices
+        - keycloakEnvironment
+        - passwordPublicKey
+      properties:
+        applicationAcronym:
+          type: string
+          example: MSSC
```

Comment:
Might be verbose, but we should always try to have a description field on every property even when it looks self evident.

Reply:
just following what was in the spec already. we know that the spec needs to be updated (endpoints that do not exist).  File this away for when the spec is updated to match production.